challenges/functions.js

```diff
@@ -1,30 +1,34 @@
-// ==== Callbacks ====  
+// ==== Callbacks ====
 
 /* Step 1: Create a callback function
   * Create a function named consume that can take 3 parameters.
   * The first two parameteres can accept any argument
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+function consume(a, b, cb) {
+  return cb(a, b);
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
-
+const add = (x, y) => x + y;
+const multiply = (x, y) => x * y;
+const greeting = (a, b) => `Hello ${a} ${b}, sup`;
 
 /* Step 3: Check your work by uncommenting the following calls to consume(): */
-//consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
-
+consume(2, 2, add); // 4
+consume(10, 16, multiply); // 160
+consume("Mary", "Poppins", greeting); // Hello Mary Poppins, nice to meet you!
 
-// ==== Closures ==== 
+// ==== Closures ====
 // Question 1: Explain in your own words why the example below is a closure.
 
-// Explanation: 
+// Explanation:
+/*Because theres acess to an internal var*/
 
 // Question 2: Given the example below, what scope is the external variable in?
 
```

Comment:
No answer here, but I'm sure that you know that the variable external is located in the Global scope.